src/test/java/org/folio/rest/impl/ReceivingHistoryTest.java

```diff
@@ -53,27 +54,28 @@ public void testReceivingHistory() {
       logger.info("--- mod-orders-storage receiving_history test: Creating Piece 2 ...");
       piecesSampleId2 = testCreatePiece(pieceSample2);
       testVerifyPieceCreated();
-      
- 
+
+
       logger.info("--- mod-orders-storage receiving_history test: Creating PoLine 1...");
       poLineSampleId = testCreatePoLine(poLineSample);
       logger.info("--- mod-orders-storage receiving_history test: Creating PoLine 2 ...");
       poLineSampleId2 = testCreatePoLine(poLineSample2);
       testVerifyPoLineCreated();
-      
-      
+
+
       logger.info("--- mod-orders-storage receiving_history test: Creating Detail 1...");
       detailSampleId = testCreateDetail(detailSample);
       logger.info("--- mod-orders-storage receiving_history test: Creating Detail 2 ...");
       detailSampleId2 = testCreateDetail(detailSample2);
       testVerifyDetailCreated();
-     
-  
+
+
       logger.info("--- mod-orders-storage pieces test: After receiving_history View created ...");
       verifyViewCollectionAfter(RECEIVING_HISTORY_ENDPOINT, 2);
 
     } catch (Exception e) {
       logger.error("--- mod-orders-storage-test: receiving_history API ERROR: " + e.getMessage(), e);
+      fail();
```

Comment:
whoops, how did this slip through the cracks before?  Please add a message to the fail call... depending on how you run the tests, it's sometimes hard to find the error message that caused the failure.